### PR TITLE
LBSD-2489 Improve Liveblog readme file and installation steps

### PR DESCRIPTION
**Pull request by dev-tsoni**
**Updates**:
- Updated Dockerfile along with bionic base image
- Updated README.md with updated installation instructions using docker

### CODE REVIEW

File: docker/common.yml [hunk 1]
```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=False
+   - SYNDICATION=True
+   - MONGO_DBNAME=superdesk
+   - SUBSCRIPTION_LEVEL=network
+   - LIVEBLOG_CACHE_TYPE=redis
+   - LIVEBLOG_CACHE_REDIS_URL=redis://redis:6379/12
```
Comment (eos87) [inline docker/common.yml:68]: Let's remove the redis cache settings, by default in local env the `simple` cache is easier approach.

File: docker/common.yml [hunk 2]
```diff
@@ -59,6 +59,18 @@ superdesk:
...
+   - BOTO_CONFIG=/doesnotexist
+   - SYNDICATION_VALIDATE_AUTH=0
+   - MAIL_SUPPRESS_SEND=False
+   - EMBEDLY_KEY=82645d4daa7742cc891c21506d28235e
```
Comment (eos87) [inline docker/common.yml:72]: I'm wondering how could we avoid passing this embody key. It's a private key in the end and we should not expose either our own key or other random key. Also I'm not sure of the legal implications. Let's better remove it to leave it somehow indicated that needs to be replaced.

File: docker/common.yml [hunk 3]
```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=False
+   - SYNDICATION=True
+   - MONGO_DBNAME=superdesk
```
Comment (eos87) [inline docker/common.yml:65]: let's change the DBNAME to liveblog instead of superdesk

File: docker/common.yml [hunk 4]
```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=False
```
Comment (eos87) [inline docker/common.yml:63]: It may be better to have LIVEBLOG_DEBUG=True.

**Comment by dev-tsoni** [comment]
Made the requested changes and changed the folder name to server instead of liveblog

File: docker/common.yml [hunk 5]
```diff
@@ -59,6 +59,18 @@ superdesk:
    - SUPERDESK_WS_URL=ws://127.0.0.1/ws
    - SUPERDESK_CLIENT_URL=http://127.0.0.1
    - SUPERDESK_TESTING=True
+   - EMBED_PROTOCOL=http://
+   - LIVEBLOG_DEBUG=True
+   - SYNDICATION=True
+   - MONGO_DBNAME=liveblog
+   - SUBSCRIPTION_LEVEL=network
+   - LIVEBLOG_CACHE_TYPE=simple
+   - LIVEBLOG_CACHE_REDIS_URL=redis://redis:6379/12
```
Comment (eos87) [inline docker/common.yml:68]: One last comment. Now that `LIVEBLOG_CACHE_TYPE=simple` we don't need the `LIVEBLOG_CACHE_REDIS_URL ` so please just remove it. Thanks

File: Dockerfile [hunk 6]
```diff
@@ -24,11 +20,10 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # setup the environment
-WORKDIR /opt/superdesk/
+WORKDIR /opt/server/
 COPY ./docker/nginx.conf /etc/nginx/nginx.conf
 COPY ./docker/superdesk_vhost.conf /etc/nginx/sites-enabled/superdesk.conf
 COPY ./docker/start.sh /opt/superdesk/start.sh
```
Comment (eos87) [inline Dockerfile:26]: @dev-tsoni will this `/opt/superdesk/start.sh` after renaming the folder to `server/`?

File: Dockerfile [hunk 7]
```diff
@@ -41,22 +36,26 @@ EXPOSE 5100
...
 # copy client sources
-COPY ./client /opt/superdesk/client
+COPY ./client ./client
 
 # TODO: this is hack to update basic themes during bamboo deployment
-COPY ./server/liveblog/themes/themes_assets/ /opt/superdesk/themes_assets/
+COPY ./server/liveblog/themes/themes_assets/ ./themes_assets/
 
-RUN cd ./client && npm i
+CMD /opt/superdesk/start.sh
```
Comment (eos87) [inline Dockerfile:61]: this should be `/opt/server/start.sh`? (because of folder rename)

File: docker/common.yml (hunk 5 above)
Comment (dev-tsoni) [reply docker/common.yml:68]: ok, will remove LIVEBLOG_CACHE_REDIS_URL

File: Dockerfile (hunk 6 above)
Comment (dev-tsoni) [reply Dockerfile:26]: ah yes, will move the start script to server

**Pull request merged**